packages/components/src/Navbar/Component.purs

```diff
@@ -180,9 +180,14 @@ customerService isPersonating activeUser = do
       userLink user =
         Router.link
```

Comment:
What about using `Router.redirect` instead? It redirects as soon as rendered IIRC.

Reply:
I had partial success with it. I could do a redirect with the personation once but if I after that return to the search page and try to select another customer, then it won't do the redirect and stays on the search page.
I'll look a bit more if I could figure out what it'd need.

Reply:
I managed to do it with the router, though it felt like I was tricking it to do it the way I wanted.